admin/user.go

```diff
@@ -211,7 +219,72 @@ func (s *Service) CreateOrganizationForUser(ctx context.Context, userID, orgName
 
 	s.Logger.Info("created org", zap.String("name", orgName), zap.String("user_id", userID))
 
-	return org, nil
+	// return early if billing is not configured
+	if s.Biller.Name() == "noop" {
+		return org, nil
+	}
```

Comment:
Not really useful to have a `noop` biller if it needs to be manually checked. Might as well remove it then and have `s.Biller == nil` checks instead.

Consider if the interface or calls can be changes such that the no-op logger accepts the same calls as the real logger. It would make test and prod code flows more similar. For example, maybe `CreateCustomer` should be refactored to automatically create a subscription with the default plan?